FibonacciNumbers/FibonacciNumbers/main.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+unsigned long long int recursiveMethod(int neededNumber){
```

Comment:
```suggestion
unsigned long long int recursiveMethod(int neededNumber) {
```
Тут и ниже. Перед открывающей фигурной скобкой всегда пробел ставится